scopesim/optics/image_plane_utils.py

```diff
@@ -239,8 +239,7 @@ def add_table_to_imagehdu(table, canvas_hdu, sub_pixel=True, wcs_suffix=""):
 
     s = wcs_suffix
     if not utils.has_needed_keywords(canvas_hdu.header, s):
-        raise ValueError("canvas_hdu must include an appropriate WCS: {}"
-                         "".format(s))
+        raise ValueError("canvas_hdu must include an appropriate WCS: {s}")
```

Comment:
```suggestion
        raise ValueError(f"canvas_hdu must include an appropriate WCS: {s}")
```

You're leaving these in intentionally to see whether I'm paying attention right :jack_o_lantern:

Reply:
Sure, absolutely xD